lib/index.js

```diff
@@ -2,9 +2,20 @@
 
 const requireIndex = require('requireindex');
 const path = require('path');
+const { name, version } = require('../package.json');
 
 module.exports.rules = requireIndex(path.join(__dirname, 'rules'));
 
+// required for eslint v9 flat config compatibility
+// https://eslint.org/docs/latest/extend/plugin-migration-flat-config
+module.exports.meta =  {
+  name,
+  version
+},
+
 module.exports.processors = {
-  '.json': require('./processors/json')
+  '.json': require('./processors/json'),
```

Comment:
leaving the `.json` for backwards compatability

Reply:
Can you add this as a comment?